### ISSUE TITLE
Jumping toolbar titles

### ISSUE THREAD

**Issue by octokim** [InflationX/Calligraphy#43]
I faced with a problem that is similar to [this one](https://github.com/chrisjenx/Calligraphy/issues/304). I have separate toolbar on each fragment and after transition toolbar's title is oddly jumping. If I set toolbar's subtitle to null the problem is gone (as in that old issue).

**Comment by ARRR42** [comment]
Same issue here

**Comment by chrisjenx** [comment]
Yes, I think thats because we force the titles to be inflated to set the typeface. Do you have a reproduceable example as I've not see this before :(
